src/Core/Services/OpenAPI/OpenApiDocumentor.cs

```diff
@@ -166,14 +167,16 @@ private OpenApiPaths BuildPaths()
         {
             OpenApiPaths pathsCollection = new();
 
-            foreach (KeyValuePair<string, DatabaseObject> entityDbMetadataMap in _metadataProvider.EntityToDatabaseObject)
+            string defaultDataSourceName = _runtimeConfig.GetDefaultDataSourceName();
```

Comment:
please create a separate tracking issue for REST Apis multiple datasource support

Reply:
Added issue:https://github.com/Azure/data-api-builder/issues/1753